File: Python_LAB_ASSIGNMENT/Assignment-9/Exception Handling/Q5.py
try :
    a = int(input("Enter first number: "))
    b = int(input("Enter second number: "))
    res = a / b
    print("The result is: ", res)
except ValueError :
    print("Invalid Input! Enter an integer!")
except ZeroDivisionError :
    print("Error! Can't be divided by zero!")

File: IWT_LAB_ASSIGNMENTS/ASSIGNMENT-5/q5.js
let arr2 = [12, 35, 1, 10, 34, 1];
let first = -Infinity, second = -Infinity;

for (let i = 0; i < arr2.length; i++) {
    if (arr2[i] > first) {
        second = first;
        first = arr2[i];
    } else if (arr2[i] > second && arr2[i] !== first) {
        second = arr2[i];
    }
}
console.log("Second Largest:", second);
